src/components/Contentful/DynamicHero.vue

```diff
@@ -76,7 +76,8 @@
 				<div class="small-10 large-6 xlarge-7 align-self-middle columns contentful-hero__cta_wrapper">
 					<h1 class="contentful-hero__header" v-html="heroHeadline">
 					</h1>
-					<div class="contentful-hero__body" v-html="heroBody">
+					<div class="contentful-hero__body">
+						<dynamic-rich-text :html="heroBody" />
```

Comment:
Example of how to use the renderer. It works with our current rich text fields, which have no embedded content